### PR TITLE
AC#7 tested, some files were added and modified

### PR DESCRIPTION
**Pull request by ghost**
I would appreciate if you can review my code and accept it.

### CODE REVIEW

**Review by huseyin9799 — APPROVED** [review]
Nice

**Review by RurouniKenshin24 — COMMENTED** [review]
Thanks

File: src/test/java/com/libraryct/step_definitions/FilterStatusDropdownStepDefinitions.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public void user_selects_from_records_dropdown(String string) {
+        Select recordsDropdown = new Select(userManagementPage.getShowRecordsDropDown());
+        recordsDropdown.selectByValue(string);
+    }
+
+    @Then("user should see all user's status are {string}")
+    public void user_should_see_all_user_s_status_are(String string) {
+        List<String> statusList = new ArrayList<>();
+        for (int i=1; i<42; i++){
+            statusList.add(Driver.getDriver().findElement(By.xpath("//tbody/tr["+i+"]/td[6]")).getText());
```
Comment (devfrango) [inline src/test/java/com/libraryct/step_definitions/FilterStatusDropdownStepDefinitions.java:36]: I'm stealing your xpath for future use. :)

**Review by devfrango — APPROVED** [review]
It's "Javaliscious."

File: src/test/java/com/libraryct/step_definitions/FilterStatusDropdownStepDefinitions.java (hunk 1 above)
Comment (ghost) [reply src/test/java/com/libraryct/step_definitions/FilterStatusDropdownStepDefinitions.java:36]: you're more than welcome man

**Pull request merged**